scripts/pre_commit_linter.py

```diff
@@ -1277,12 +1392,12 @@ def _check_sorted_dependencies(self):
                         if sorted_imports != literal_args:
                             failed = True
                             print (
-                                'Please ensure that in %s in file %s, the '
-                                'stringfied dependencies should be in the '
-                                'following manner: dollar imports, regular '
-                                'imports and constant imports, all in sorted '
-                                'order.'
-                                % (property_value, filepath))
+           	                'Please ensure that in %s in file %s, the '
```

Comment:
Hi @seanlip I managed to fix the above lines I can't seem to fix these. I tried tweaking it around, used different editors and also changed back and forth between spaces and tabs.

Reply:
Ah ok. I think you might need to either ask @oppia/dev-workflow-team for help in this case, or file a support issue with GitHub.

Reply:
Hi @oppia/dev-workflow-team I can't seem to fix these spaces in my local env. Any thoughts? I tried changing editors and fixing spaces and tabs. @seanlip Should I just fix these by the online editor?

Reply:
Hi @lilithxxx, I wouldn't recommend using the online editor here. There'll still be this discrepancy between your local workspace and GitHub that is just going to continue through future PRs, so we should get to the bottom of it.

One way you can try to debug is to mark this PR as WIP for a bit, and just make a bunch of commits to it so that you can understand how the spaces on your local IDE correspond with the ones that show on the Files Changed tab (feel free to temporarily comment out the linter script if needed). Maybe also write a short script to check that the characters in your file actually are regular spaces and not some other whitespace character (like tabs) that may have different lengths in different displays. But I think it's important to figure out what is actually going on (and, if necessary, make a fix for it) so that this doesn't happen in the future.

Note that the "Files Changed" version is canonical and you should optimize for that (i.e., think about how to get your IDE to match the "Files Changed" version rather than the other way around).

Reply:
Hi @seanlip I made a script to detect spaces and found that there was indeed a tab hidden there. PTAL